components/tidb_query/src/expr/ctx.rs

```diff
@@ -244,7 +244,7 @@ impl EvalContext {
         if !is_overflow {
             Ok(())
         } else {
-            self.handle_overflow_err(Error::overflow("DECIMAL", ""))
+            self.handle_overflow_err(Error::overflow("", ""))
```

Comment:
It's a problem indeed that `handle_overflow` only works for Decimal but I think we should change the function name or remove the function.